Enhancements in Help.vue FAQs section

abhisek-1221 opened the pull request:
The PR resolves the issue #345 . Here's a brief look into it with the changes made for the responsive design.

> Before 

https://github.com/ruxailab/RUXAILAB/assets/110292494/3cfb3da8-f7f5-44cf-ab8f-6f86ec537894

> After

https://github.com/ruxailab/RUXAILAB/assets/110292494/47c937d9-4ad0-417b-bacb-553f67dd929a

jvJUCA reviewed (approved):
Congratulations! 🎉 Your pull request has been accepted! Thank you for your valuable contribution.

Your changes have been merged successfully and will be included in the develop branch. Your effort in improving our project is greatly appreciated.

Thanks again for your contribution, and we look forward to your continued involvement in our project!

Best regards,
@jvJUCA

jvJUCA commented:
It would also be highly appreciated to have a pull request adding GIFs addressing the issues mentioned in the FAQ. This comes right after accepting a pull request from the person who fixed the FAQ's responsiveness.

Best regards,

@jvJUCA

abhisek-1221 commented:
> Dear @abhisek-1221,
> 
> It would also be highly appreciated to have a pull request adding GIFs addressing the issues mentioned in the FAQ.
> 
> Best regards,
> 
> @jvJUCA

I'm working on it !
